Cargo.toml

```diff
@@ -38,6 +38,7 @@ frame-metadata = "12.0.1"
 frame-support = "2.0.1"
 sp-runtime = "2.0.1"
 sp-version = "2.0.1"
+sp-finality-grandpa = { version = "2.0.1", default-features = false }
```

Comment:
I'd like to avoid adding this dependency because it is very heavy, I have removed them in a previous PR https://github.com/paritytech/substrate-subxt/pull/194 by defining types locally.

It looks like we re just using `AuthorityList` which is `pub type AuthorityList = Vec<(AuthorityId, AuthorityWeight)>;`. For `AuthorityId` you should be able to use the locally defined one, and AuthorityWeight is just a `u64`

Reply:
Thanks. It makes sense to me. I've defined them locally.